sql/telemetry_derived/devtools_accessiblility_panel_usage_v1/query.sql

```diff
@@ -0,0 +1,172 @@
+        "browser-context-menu"
+      )
+    ) AS browser_context_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_select_accessible_for_node,
+        "inspector-context-menu"
+      )
+    ) AS inspector_context_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_audit_activated,
+        "ALL"
+      )
+    ) AS audit_all_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_audit_activated,
+        "CONTRAST"
+      )
+    ) AS audit_contrast_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_audit_activated,
+        "KEYBOARD"
+      )
+    ) AS audit_keyboard_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_audit_activated,
+        "TEXT_LABEL"
+      )
+    ) AS audit_text_label_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_accessible_context_menu_item_activated,
+        "print-to-json"
+      )
+    ) AS print_to_json_context_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+        "accessibility.simulation.deuteranomaly"
+      )
+    ) AS simulation_deuteranomaly_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+        "accessibility.simulation.protanomaly"
+      )
+    ) AS simulation_protanomaly_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+        "accessibility.simulation.protanopia"
+      )
+    ) AS simulation_protanopia_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+        "accessibility.simulation.deuteranopia"
+      )
+    ) AS simulation_deuteranopia_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+        "accessibility.simulation.tritanopia"
+      )
+    ) AS simulation_tritanopia_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+        "accessibility.simulation.tritanomaly"
+      )
+    ) AS simulation_tritanomaly_count,
+    SUM(
+      udf.get_key(
+        payload.processes.parent.keyed_scalars.devtools_accessibility_simulation_activated,
+        "accessibility.simulation.contrastLoss"
+      )
+    ) AS simulation_contrast_loss_count
+  FROM
+    telemetry.main
+  WHERE
+    sample_id = 0
+    AND DATE(submission_timestamp) = @submission_date
+    AND payload.processes.parent.scalars.devtools_accessibility_opened_count > 0
+  GROUP BY
+    1,
+    2,
+    3
```

Comment:
```suggestion
    submission_date,
    client_id,
    normalized_channel
```

We consider it good practice to name grouping columns explicitly to avoid unintentional changes when reordering columns.